### PR TITLE
Refactor Embeds to use custom components

### PR DESCRIPTION
**Pull request by mattwr18**
### Todo

- [X] None

### CODE REVIEW

File: backend/src/schema/resolvers/embeds.js [hunk 1]
```diff
@@ -11,16 +11,17 @@ export default {
...
       'publisher',
-      'date',
-      'description',
       'url',
+      'date',
       'image',
       'audio',
       'video',
       'lang',
-      'html',
```
Comment (roschaefer) [inline backend/src/schema/resolvers/embeds.js]: Hm, I think for the trusted oembed providers in `providers.json` we can return plain html to the client.

File: webapp/components/Embeds/VideoEmbed.vue [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    },
+  },
+  data() {
+    return {
+      renderEmbed: false,
+    }
+  },
+  computed: {
+    embedUrl() {
+      if (this.embedData.publisher === 'YouTube') {
```
Comment (roschaefer) [inline webapp/components/Embeds/VideoEmbed.vue]: No, I don't want to see this kind of complexity. I would prefer to stick to the current solution for more complex embeds. 

 For security, have a look if we can trust the oembed provider. For data privacy, wrap the embed with an additional click (are you OK to load stuff from 3rd parties?). For mobile responsiveness, try out if the iframe or whatever breaks the layout.

File: webapp/components/Editor/nodes/Embed.spec.js [hunk 3]
```diff
@@ -9,23 +9,17 @@ describe('Embed.vue', () => {
   beforeEach(() => {
```
Comment (roschaefer) [inline webapp/components/Editor/nodes/Embed.spec.js]: I love that you created test cases for the embeds :heart:

**Comment by mattwr18** [comment]
closing this in favor of https://github.com/Human-Connection/Human-Connection/pull/1686
we could maybe still refactor in the future to have separate components based on what is returned from `oEmbed`...

**Pull request closed without merging**